fvtest/tril/test/CompileTest.cpp

```diff
@@ -56,3 +56,37 @@ TEST_F(CompileTest, NoCodeGen) {
 
     EXPECT_TRUE(verifier.hasRun());
 }
+
+TEST_F(CompileTest, Ladd) {
+    auto trees = parseString("(method name=\"ladd\" return=\"Int64\" args=[\"Int64\", \"Int64\"] (block (ireturn (ladd (lload parm=0) (lload parm=1)))))");
+
+    ASSERT_NOTNULL(trees);
+
+    Tril::DefaultCompiler compiler{trees};
+
+    ASSERT_EQ(0, compiler.compile()) << "Compilation failed";
+
+    auto entry = compiler.getEntryPoint<int32_t (*)(int)>();
```

Comment:
`getEntryPoint<int32_t (*)(int)>()` should be `getEntryPoint<int64_t (*)(int64_t, int64_t)>()` when the compiled method takes two `Int64` arguments and returns an `Int64`.